### ISSUE TITLE
Weird \n in output. 

### ISSUE THREAD

**Issue by eduxstad** [Jythonscript/DS_Store-count#1]
Running the latest version I get this output. 

```8 DS_Store files were found in /home/mathboy19
They add up to 68K in size
\nWould you like to see where they are?
[y/n]: y
```
With the weird \n. Pls fix thank you.

**Comment by Jythonscript** [comment]
It should be fixed now, as of [](https://github.com/Jythonscript/DS_Store-count/commit/3921edeade5712288adb5b2b54cf9718e1ccd833)

**Comment by Jythonscript** [comment]
It should be fixed now, as of [https://github.com/Jythonscript/DS_Store-count/commit/3921edeade5712288adb5b2b54cf9718e1ccd833](url)

**Comment by Jythonscript** [comment]
It should be fixed now, as of commit 3921ede Please test it to make sure.
